src/pybind/cephfs/cephfs.pyx

```diff
@@ -238,9 +239,9 @@ cdef make_ex(ret, msg):
     """
     ret = abs(ret)
     if ret in errno_to_exception:
-        return errno_to_exception[ret](msg)
+        return errno_to_exception[ret](ret, msg)
```

Comment:
this is on purpose. i am trying to model after `OSError` in py 3.3+. and IMHO, `(22, 'an error occurred')` is easier to read for a developer, than `an error occurred: error code 22`. the reason i want to have a better error message, is for ease of debugging. ideally, we should catch an exception and handle it properly instead of printing it directly. so i think it's okay.